lib/bb/generate_report_request_form.rb

```diff
@@ -25,15 +25,22 @@ class GenerateReportRequestForm < Common::Form
     # MHV error is preferable.
     # validates :from_date, date: { before: :to_date, message: 'must be before to date' }
     validates :data_classes, presence: true
-    validate  :data_classes_belongs_to_eligible_data_classes
+    # TODO: temporary hack to make eligible data classes work
```

Comment:
I guess I'd rather see a link to the GH issue where the longer-term fix is proposed instead of the PR.